cmd/minersc.go

```diff
@@ -184,10 +184,34 @@ var minerscMiners = &cobra.Command{
 			cb    = NewJSONInfoCB(info)
 		)
 
-		if err = zcncore.GetMiners(cb); err != nil {
-			log.Fatal(err)
+		fmt.Println("info here :::::::::::::::::::::::::", info)
+
+		limit, offset := 20, 0
```

Comment:
Let add `--all` flag to `ls-miners`, and iterate to get all.

Reply:
In such a case, what to do about limit and offset if passed?

Reply:
for all you need to laod in iterator manner https://github.com/0chain/zboxcli/pull/359